applications/DEM_application/custom_elements/spheric_particle.h

```diff
@@ -280,14 +282,15 @@ std::vector<array_1d<double, 3> > mNeighbourElasticExtraContactForces;
 
 virtual void ComputeAdditionalForces(array_1d<double, 3>& externally_applied_force, array_1d<double, 3>& externally_applied_moment, const ProcessInfo& r_process_info, const array_1d<double,3>& gravity);
 virtual array_1d<double,3> ComputeWeight(const array_1d<double,3>& gravity, const ProcessInfo& r_process_info);
+virtual void CalculateOnContactElements(size_t i_neighbour_count, double LocalContactForce[3]);
 
 array_1d<double, 3> mContactMoment; //SLS
 
 Matrix* mStressTensor;
 Matrix* mSymmStressTensor;
 double mPartialRepresentativeVolume;
 
-
+unsigned int mNeighborsSize;
```

Comment:
Can we avoid this variable by using `mNeighbors.size()` ?

Reply:
Of course we can, good point @maceligueta